Content.Server/Chemistry/ReactionEffects/AreaReactionEffect.cs

```diff
@@ -47,6 +47,7 @@ public abstract class AreaReactionEffect : ReagentEffect, ISerializationHooks
         /// <summary>
         /// Used to calculate concentration. Reagents get linearly more concentrated as the range goes from
         /// _reagentDilutionStart to zero. When the range is zero the reagents volume gets multiplied by this.
+        /// TODO: This appears to be unused, but maybe should be used, need to investigate
```

Comment:
git blame this & ping the corresponding person tbh

Reply:
I bugged Radrark#1226 on discord

Reply:
Mirrorcult says we don't need it